.storybook/main.js

```diff
@@ -27,7 +27,6 @@ module.exports = {
     // monorepo.
     // See https://storybook.js.org/docs/react/configure/webpack
     webpackFinal: async (config, {configType}) => {
-        const wbPkgJsonRegex = /^@khanacademy\/wonder-blocks(-.*)\/package.json$/;
```

Comment:
I removed this style of imports a while ago but forgot to clean this up so I'm doing so now.